src/07_slices.py

```diff
@@ -12,26 +12,33 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+x = slice(1,2)
+print(a[x])
 
```

Comment:
This is good!

Note that the exercise asks you to use slice notation. Since you understand how to use the `slice()` method, your just a step away from using the notation to make your solution more terse:

`x = a[1:2]`

Just take the range your passing into `slice(1,2)` and plug it right into the brackets separated by a colon.

You'll see/use slice-notation in the future so take a quick look at this when you have the chance: https://stackoverflow.com/a/509295